cloud/disk_manager/test/acceptance/test_runner/lib/instance_policy.py

```diff
@@ -62,6 +62,8 @@ def obtain(self) -> Ycp.Instance:
                                          f' {self._zone_id}')
         try:
             yield self._instance
+        except Exception:
```

Comment:
You do not need to re-raise an exception.It will be still raised, yet another line will be added to the stack trace.

Reply:
It doesn't look like it will be risen again if you omit raise instruction

Reply:
My point was that the block 
```
except Exception:
    raise
```
Is the same is no `except` block at the moment. No that note is pointless though.